.travis.yml

```diff
@@ -1,13 +1,32 @@
 language: cpp
 compiler: gcc
+dist: trusty
 
-dist: xenial
 addons:
   apt:
+    sources:
+    - sourceline: 'ppa:beineri/opt-qt593-trusty'
```

Comment:
Since Qt will be part of the AppImage anyway and `beineri` provides Qt 5.10 for Trusty, why not just the newer version?

Reply:
Ah, I guess because 5.9 is LTS and 5.10 is not?

Reply:
You decide, if you think 5.10 provides a benefit we can switch to that.

Reply:
No, I guess LTS is the sane choice.